### PR TITLE
[WIP] Add ability to delete a single event id

### PR DESCRIPTION
**Pull request by mattrobenolt**
@getsentry/api 

@dcramer you mentioned there'd be work that needs to be done? So I'm open to changing this, but couldn't tell what this would harm.

---

This change is [Reviewable](https://reviewable.io/reviews/getsentry/sentry/3465)

### CODE REVIEW

File: src/sentry/api/endpoints/event_details.py [hunk 1]
```diff
@@ -116,3 +118,23 @@ def get(self, request, event_id):
...
+        Remove an Event
+        ```````````````
+
+        Removes an individual Event.
+
+        :pparam string event_id: the ID of the event to delete.
+        :auth: required
+        """
+        from sentry.tasks.deletion import delete_events
+        delete_events(relation={'id': event.id}, limit=1)
```
Comment (dcramer) [inline src/sentry/api/endpoints/event_details.py]: do not call this synchronously

**Comment by dcramer** [comment]
delete_events does not attempt to clean up no longer valid tag links on an issue. This is fine because of how its used historically, but when removing an individual event that needs to happen.

**Comment by mattrobenolt** [comment]
Yeah, I get it now. Another case would be, if there's only one event in the group, we'd also need to delete the Group since having a group with 0 events will cause other issues.

Cool, thanks, just wanted to throw this up quickly to figure out what more would need to be done.

**Comment by mitsuhiko** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/3465)*

**Comment by markstory** [comment]
This pull request cannot be merged currently. Without further updates it will be closed in a few days.

**Pull request closed without merging**